easybuild/easyconfigs/p/Pydusa/Pydusa-20180515_numpy_header_path.patch

```diff
@@ -0,0 +1,14 @@
+Allow defining a custom numpy header path with environment variable PY_HEADER_NUMPY
+Author: Samuel Moors, Vrije Universiteit Brussel (VUB)
+diff -ur pydusa-20180515.orig/configure pydusa-20180515/configure
+--- pydusa-20180515.orig/configure	2018-05-15 17:01:43.000000000 +0200
++++ pydusa-20180515/configure	2018-07-02 11:19:06.219901276 +0200
+@@ -5097,6 +5097,8 @@
+    PY_HEADER_NUMPY="-I${PY_PREFIX}/Extras/lib/python/numpy/core/include"
+ elif test -d ${PY_PREFIX}/lib/python$PY_VERSION/site-packages/numpy/core/include; then 
+    PY_HEADER_NUMPY="-I${PY_PREFIX}/lib/python$PY_VERSION/site-packages/numpy/core/include"   
++elif test -d $PY_HEADER_NUMPY; then
++   PY_HEADER_NUMPY="-I$PY_HEADER_NUMPY"
```

Comment:
@smoors Is this patch scrictly required? Why not use `$PY_PREFIX` and `$PY_VERSION` instead?

Reply:
Sorry, I misunderstood this I guess, this is about letting the `configure` script pick up on `PY_HEADER_NUMPY` itself...